Autopilot/Inc/airspeed.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+*   
+*   Airspeed sensor functions
+*
+*   This header is an outline of basic functions of the airspeed
+*   sensor. Generally, this sensor is a Pitot tube which uses a 
+*   pressure differential to determine airspeed. 
+*
+*   Airspeed calculation should be done and returned in meters 
+*   per second
+*
+*    Author: Annie LePage
+**/
+
+
+#ifndef AIRSPEED_HPP
+#define AIRSPEED_HPP
+
+#include <stdint.h>
```

Comment:
I'm not a fan of including libraries that don't need to be there. It doesn't look like you need the stdint library in this file.

Reply:
You're right, I had thought about using integers and then decided against